SharedPackages/BrowserServicesKit/Sources/Subscription/Managers/SubscriptionManagerV2.swift

```diff
@@ -551,6 +556,10 @@ public final class DefaultSubscriptionManagerV2: SubscriptionManagerV2 {
 
     public func signOut(notifyUI: Bool) async {
         Logger.subscription.log("SignOut: Removing all traces of the subscription and account. Notify UI: \(notifyUI ? "true" : "false")")
+
+        if notifyUI { userInitiatedSignOut = true }
```

Comment:
I'm relying on `notifyUI` to determine whether we want to show the prompt. It's not only set when the user initiated the sign out, but the other cases are less likely to occur when the VPN is installed.